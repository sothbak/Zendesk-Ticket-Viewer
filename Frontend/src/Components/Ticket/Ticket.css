li { 
    cursor: pointer;
    list-style-type: none;
}

li:hover {
    background-color: rgb(143, 184, 184);
}