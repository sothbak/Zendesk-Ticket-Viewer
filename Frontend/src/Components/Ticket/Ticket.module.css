.ticket {
    cursor: pointer;
    list-style-type: none;
    border: 2px black;
    margin-left: 20px;
    margin-right: 50px;
    padding: 5px;
    border:1px solid #000;
}

.ticket:hover {
    background-color: rgb(143, 184, 184);
}