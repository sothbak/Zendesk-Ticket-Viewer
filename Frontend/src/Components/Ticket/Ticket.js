import React, { Component } from 'react';
import './Ticket.css';

class Tickets extends Component {

    handleClick = () => {
        this.props.clickDisplayTicket(this.props.id, this.props.subject, this.props.status, this.props.description);
    }
    handleClick = this.handleClick.bind(this);

    render() {
        return (
            <li onClick={this.handleClick}>
                <h1>{this.props.subject}</h1>
                <p>{this.props.created_at}</p>
            </li>
        )
    }
}

export default Tickets;