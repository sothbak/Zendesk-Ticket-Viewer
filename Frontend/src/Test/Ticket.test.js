import React from 'react';
import { shallow } from 'enzyme';
import Ticket from '../Components/Ticket/Ticket.js';
describe('Ticket', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Ticket />);
    expect(wrapper).toMatchSnapshot();
  });

});