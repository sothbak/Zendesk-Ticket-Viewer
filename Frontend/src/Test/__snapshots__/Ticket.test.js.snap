// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticket should render correctly 1`] = `
<li
  onClick={[Function]}
>
  <h1 />
  <p />
</li>
`;
